core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -63,15 +70,8 @@ public static MetricsConfig getDefault() {
     return DEFAULT;
   }
 
-  /**
-   * Creates a metrics config from table configuration.
-   *
-   * @param props table configuration
-   * @deprecated use {@link MetricsConfig#forTable(Table)}
-   */
-  @Deprecated
-  public static MetricsConfig fromProperties(Map<String, String> props) {
-    return from(props, null, null);
+  public static MetricsConfig getDefaultForPosDelete() {
```

Comment:
nit: remove `get` from the name. See: https://iceberg.apache.org/contribute/#method-naming

Reply:
Should we also replace the `getDefault()` with `default()`

Reply:
nah.. good point. If we already have a `getDefault`, we can't change that, and we should conform. This means that your original version was better. Sorry for the back and forth